tests/test_utils.py

```diff
@@ -130,6 +130,17 @@ def test_short_circuit_dict(self):
 
         self.assertEqual(Accessor("occupation__name").resolve(context), "Carpenter")
 
+    def test_callable_args_kwargs(self):
+        class MyClass:
+            def method(self, *args, **kwargs):
+                return args, kwargs
+
+        callable_args = ("arg1", "arg2")
+        callable_kwargs = {"kwarg1": "val1", "kwarg2": "val2"}
+        obj = MyClass()
+        result = Accessor("method", *callable_args, **callable_kwargs).resolve(obj)
+        self.assertEqual(result, (callable_args, callable_kwargs))
```

Comment:
Next step: supporting `Accessor` as `arg`/`kwarg` to support even more dynamic constructs 😬